docs/application-connector/08-03-register-manage-services.md

```diff
@@ -202,17 +204,18 @@ This is an example of the `api` section of the request body for an API with a sp
                 "clientSecret": "{CLIENT_SECRET}"
             },
         }  
+    }
 ```
 
 ## Use custom headers and query parameters for fetching API specification from URL 
 
-You can specify additional headers and query parameters that will be injected to requests to the specification URL.
+You can specify additional headers and query parameters to inject to requests to the specification URL.
```

Comment:
Sooo many `to` in here. It sounds a bit weird. Can we make it sound more natural?